### PR TITLE
Replaced offsets with array of vertices

### PR DESCRIPTION
**Pull request by NaveTK**
Accidently hit Ctrl+T so it added some spaces or linebreaks at various positions of those files and marked them as changed, I hope that's not that huge of a deal, other than that all I did was replace the offset array with an array of the vertices directly which we then use to render and check collisions of the asteroid without recalculating them every time

### CODE REVIEW

**Comment by MunWolf** [comment]
Could you please make this.r equal to the maximum radius of the asteroid or change the vertex generation so it doesn't go over this.r as it can be used to make collision more efficent? (I never liked how the asteroid could get bigger then the specified radius)

**Comment by NaveTK** [comment]
kk, fixed those things

**Review by MunWolf — APPROVED** [review]
Approved from me.

**Review by JonathanHHenson — APPROVED** [review]
Looks good now

File: laser.js [hunk 1]
```diff
@@ -37,9 +57,9 @@ function Laser(world, params) {
     var last_pos = p5.Vector.sub(this.pos, this.vel);
     var last_angle = p5.Vector.sub(last_pos, entity.pos).heading();
     var new_angle = p5.Vector.sub(this.pos, entity.pos).heading();
-    var asteroid_vertices = entity.vertices(last_angle, new_angle);
+    var asteroid_vertices = entity.globalVertices();
```
Comment (MunWolf) [inline laser.js:58]: last_angle and new_angle are not used anymore so please delete them here.

Comment (NaveTK) [reply laser.js:58]: oh right, you didn't use them anywhere else, one second please

**Review by MunWolf — APPROVED** [review]
Looks good now.

**Pull request merged**